start-shapleq.sh

```diff
@@ -2,17 +2,7 @@
 
 set -u
 
-zkhost=$1
-zkport=$2
-configpath=$3
-res=$( echo ruok | nc $zkhost $zkport )
-
-
-while [ "$res" != "imok" ]
-do
-  echo "waiting for zookeeper run" 
-  sleep 1
-  res=$(echo ruok | nc $zkhost $zkport)
-done
+zkquorum=$1
+configpath=$2
```

Comment:
Is there no need to add hostname as an argument?

Reply:
The script file is existed only for compatibility.

Therefore, We need to change docker process for general purpose. I think that its work is not focused on this PR. To discussion of its work, what is the docker process for general purpose, then the script file is changed.